pontoon/insights/utils.py

```diff
@@ -460,6 +460,6 @@ def get_global_pretranslation_quality(category, id):
     ]
 
     return {
-        "dates": list({convert_to_unix_time(x["month"]) for x in actions}),
+        "dates": sorted(list({convert_to_unix_time(x["month"]) for x in actions})),
```

Comment:
This is already merged, so it shouldn't appear in the diff:
https://github.com/mozilla/pontoon/commit/136e382a1137ae748d0668ed406b04538d30e709